motoman_sda10f_support/urdf/sda10f_macro.xacro

```diff
@@ -6,13 +6,13 @@
         
         <xacro:torso name="${prefix}torso" prefix="${prefix}torso_"/>
 
-        
-        <xacro:motoman_arm name="${prefix}arm_left" prefix="${prefix}arm_left_" parent="${prefix}torso_link_b1" reflect="1">
-            <origin xyz="0.09996 0.0275 0.32214" rpy="1.57 0 0" />
-        </xacro:motoman_arm>      
-        
-        <xacro:motoman_arm name="${prefix}arm_right" prefix="${prefix}arm_right_" parent="${prefix}torso_link_b1" reflect="-1">
-            <origin xyz="0.09996 -0.0275 0.32214" rpy="1.57 0 0" />
+
+        <xacro:motoman_arm name="${prefix}arm_left" prefix="${prefix}arm_left_" parent="${prefix}torso_link_b1" reflect="1" joint_limited="false">
```

Comment:
Is the `joint_limited` a copy-pasta? It's not an argument supported by the `motoman_arm` macro afaict.

Reply:
sorry I messed up with some other branches

Reply:
So you have a `joint_limited` version of the urdf?